### PR TITLE
dev/implement-step-up-token-for-inviting-au

### PR DESCRIPTION
**Pull request by PeterThrels**
redirect to SCA Page

### CODE REVIEW

**Review by PeterThrels — COMMENTED** [review]
Still Testing

File: pages/users/add.vue [hunk 1]
```diff
@@ -63,13 +63,15 @@
...
 import { email, maxLength, required } from 'vuelidate/lib/validators'
-import BaseMixin from '~/mixins/BaseMixin'
 import { CreateUserRequestModel } from '~/plugins/weavr-multi/api/models/users/requests/CreateUserRequestModel'
 import { UserModel } from '~/plugins/weavr-multi/api/models/users/models/UserModel'
-import ValidationMixin from '~/mixins/ValidationMixin'
 import { Nullable } from '~/global'
+import BaseMixin from '~/mixins/BaseMixin'
+import ValidationMixin from '~/mixins/ValidationMixin'
+
+const Cookie = process.client ? require('js-cookie') : undefined
```
Comment (oleggrech-t) [inline pages/users/add.vue:74]: Can we maybe globalize this or have it a mixin?

Comment (PeterThrels) [reply pages/users/add.vue:74]: > Can we maybe globalize this or have it a mixin?

we have this in other places, maybe could be improvement in another ticket?

**Review by jasnvella2 — APPROVED** [review]

File: pages/users/add.vue (hunk 1 above)
Comment (oleggrech-t) [reply pages/users/add.vue:74]: Sure

**Pull request closed without merging**